airbyte-integrations/connectors/source-mixpanel/source_mixpanel/source.py

```diff
@@ -699,7 +699,7 @@ class Export(DateSlicesMixin, IncrementalMixpanelStream):
 
     primary_key: str = None
     cursor_field: str = "time"
-    reqs_per_hour_limit: str = 60  # 1 query per minute
+    reqs_per_hour_limit: int = 60  # 1 query per minute
```

Comment:
Since `IncrementalMixpanelStream` extends `MixpanelStream` and now this value is equal to `MixpanelStream#reqs_per_hour_limit`, I think you don't have to have this property here.

Reply:
agreed & removed.